.storybook/stories/GlobalPromoBanner.stories.js

```diff
@@ -0,0 +1,69 @@
+import Vue from 'vue'
+import StoryRouter from 'storybook-vue-router';
+import GlobalPromoContentful from '@/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful';
+
+// import plugins
+import kivaPlugins from '@/plugins';
+Vue.use(kivaPlugins)
+
+let mockedContentfulAPIResponse = {
```

Comment:
Nitpick, should this be a const instead?

Reply:
👍